src/resolvers/Mutation/division.js

```diff
@@ -0,0 +1,26 @@
+const division = {
+  async createDivision(parent, { name, description }, ctx, info) {
+    return ctx.db.mutation.createDivision({
+      data: { name, description },
+      info
+    })
+  },
+
+  async updateDivision(parent, { id, name, description }, ctx, info) {
+    return ctx.db.mutation.updateDivision({
+      where: { id },
+      data: { name, description },
+      info
+    })
+  },
+
+  async deleteDivision(parent, { id }, ctx, info) {
+    return ctx.db.mutation.deleteDivision({ where: { id }})
+  }
+
+};
+
+module.exports = {
+  division
+};
+
```

Comment:
if you decide to split multiple mutations, it's better to rename folder Mutation to Mutations, and inside of it, index.js file should be created

Reply:
@vagabondhoang I just follow to the boilerplate that Prisma gives us:
https://github.com/graphql-boilerplates
If the project is growing up, we will split multiple later on.

Reply:
It's fine.